testnets/base_allocations.csv

```diff
@@ -1,72 +1,82 @@
 "amount","denom","address"
+"10_000_000__000_000","upenumbra","penumbrav2t1xx7me5h63ah9fkncd9ult0r6ln4ntwkf57u3g0yax9h7qf465tlrmvpwq9eulr0gugqr3aqucgny2plm336am5pz0u2clpjeqjfd5hn4fxxlzya7zs69qlqqg6fm726jvx9d4g"
+"10_000_000__000_000","gm","penumbrav2t1xx7me5h63ah9fkncd9ult0r6ln4ntwkf57u3g0yax9h7qf465tlrmvpwq9eulr0gugqr3aqucgny2plm336am5pz0u2clpjeqjfd5hn4fxxlzya7zs69qlqqg6fm726jvx9d4g"
+"10_000_000__000_000","gn","penumbrav2t1xx7me5h63ah9fkncd9ult0r6ln4ntwkf57u3g0yax9h7qf465tlrmvpwq9eulr0gugqr3aqucgny2plm336am5pz0u2clpjeqjfd5hn4fxxlzya7zs69qlqqg6fm726jvx9d4g"
+"10_000_000__000_000","pizza","penumbrav2t1xx7me5h63ah9fkncd9ult0r6ln4ntwkf57u3g0yax9h7qf465tlrmvpwq9eulr0gugqr3aqucgny2plm336am5pz0u2clpjeqjfd5hn4fxxlzya7zs69qlqqg6fm726jvx9d4g"
+"10_000_000__000_000","cube","penumbrav2t1xx7me5h63ah9fkncd9ult0r6ln4ntwkf57u3g0yax9h7qf465tlrmvpwq9eulr0gugqr3aqucgny2plm336am5pz0u2clpjeqjfd5hn4fxxlzya7zs69qlqqg6fm726jvx9d4g"
+"10_000_000__000_000","test_usd","penumbrav2t1xx7me5h63ah9fkncd9ult0r6ln4ntwkf57u3g0yax9h7qf465tlrmvpwq9eulr0gugqr3aqucgny2plm336am5pz0u2clpjeqjfd5hn4fxxlzya7zs69qlqqg6fm726jvx9d4g"
+"10_000_000__000_000","test_btc","penumbrav2t1xx7me5h63ah9fkncd9ult0r6ln4ntwkf57u3g0yax9h7qf465tlrmvpwq9eulr0gugqr3aqucgny2plm336am5pz0u2clpjeqjfd5hn4fxxlzya7zs69qlqqg6fm726jvx9d4g"
+"10_000_000__000_000","test_eth","penumbrav2t1xx7me5h63ah9fkncd9ult0r6ln4ntwkf57u3g0yax9h7qf465tlrmvpwq9eulr0gugqr3aqucgny2plm336am5pz0u2clpjeqjfd5hn4fxxlzya7zs69qlqqg6fm726jvx9d4g"
+"10_000_000__000_000","test_atom","penumbrav2t1xx7me5h63ah9fkncd9ult0r6ln4ntwkf57u3g0yax9h7qf465tlrmvpwq9eulr0gugqr3aqucgny2plm336am5pz0u2clpjeqjfd5hn4fxxlzya7zs69qlqqg6fm726jvx9d4g"
+"10_000_000__000_000","test_osmo","penumbrav2t1xx7me5h63ah9fkncd9ult0r6ln4ntwkf57u3g0yax9h7qf465tlrmvpwq9eulr0gugqr3aqucgny2plm336am5pz0u2clpjeqjfd5hn4fxxlzya7zs69qlqqg6fm726jvx9d4g"
```

Comment:
I think these should drop the `__000_000`, which is pricing in the base/display factor, since the allocations are using the display units anyways.